crates/egui_plot/src/lib.rs

```diff
@@ -1666,7 +1682,7 @@ impl<'a> PreparedPlot<'a> {
             return (Vec::new(), None);
         }
 
-        let interact_radius_sq = (16.0_f32).powi(2);
+        let interact_radius_sq = self.interact_distance.powi(2);
```

Comment:
It seems to me we should instead just use [`ui.style().interact.interact_radius`](https://docs.rs/egui/latest/egui/style/struct.Interaction.html#structfield.interact_radius) here. One global setting, instead of having to set it on each plot